sway/sway-bar.5.scd

```diff
@@ -50,6 +50,10 @@ Sway allows configuring swaybar in the sway configuration file.
 *workspace\_buttons* yes|no
 	Enables or disables workspace buttons on the bar. Default is _yes_.
 
+*strip\_workspace\_name* yes|no
+	If set to _yes_, then workspace name will be omitted from the workspace
```

Comment:
"then workspace name" is grammatically incorrect. Sucks because the i3 devs should have named it `strip_workspace_names` but that's what we got.
I'm also not sure about saying "custom number" either. Maybe just "number".